include/robot_localization/navsat_conversions.hpp

```diff
@@ -204,13 +204,17 @@ static inline char UTMLetterDesignator(double Lat)
  * East Longitudes are positive, West longitudes are negative.
  * North latitudes are positive, South latitudes are negative
  * Lat and Long are in fractional degrees
+ * Meridian convergence is computed as for Spherical Transverse Mercator,
+ * which gives quite good approximation.
+ *
+ * @param[out] gamma meridian convergence at point (degrees).
  *
  * Written by Chuck Gantz- chuck.gantz@globalstar.com
  */
 static inline void LLtoUTM(
   const double Lat, const double Long,
   double & UTMNorthing, double & UTMEasting,
-  std::string & UTMZone)
+  std::string & UTMZone, double & gamma)
```

Comment:
[Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Output_Parameters) (ROS2-recommended guidelines) says that an output parameter like this should be a pointer to a non-`const` rather than a reference. This should be true for ROS1 as well. I know this is a bigger issue that I'll address in a different ticket but the question is: Are we treating the ROS2 version of this package as an extension of the ROS1 package (meaning I should create the MR against ROS1) or treating them separately but cross-applying MRs that make sense (meaning I should create two MRs - one for each version)?

Reply:
I think that if there's other instances of this, we shouldn't block a cherry-pick PR from this codebase.

Reply:
I agree, I just wanted clarification on how the ROS1/ROS2 thing was being handled to know where to file the MR. Not intending to hold approval just for this.

Reply:
Pedantic, but IMO references are fine and are preferred, google doesn't even follow google's style guide in many cases. I avoid passing by pointers where feasible, I think that creates opportunity for people not used to pointers to royally mess up. Readability / understand-ability to the largest audience reasonable is king.

Reply:
This header comes from another package whose standards don't match the rest of the package anyway. If we want to do a PR later to bring everything in line with standards, that's fine, but I'm fine with leaving this as-is for now.